### PR TITLE
Rename instances of GoldRush-Edit to GoldPolish

### PR DESCRIPTION
**Pull request by jwcodee**
Conference title is left as is.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ GoldRush iterates through the input long reads to produce a "golden path" of rea
 
 ### General steps in the algorithm:
 1. **GoldPath** (aka GoldRush-Path): selecting the golden path reads
-2. **[GoldPolish](https://github.com/bcgsc/goldpolish)** (aka GoldRush-Edit): polishing the genome
+2. **[GoldPolish](https://github.com/bcgsc/goldpolish)**: polishing the genome
```
Comment (lcoombe) [inline README.md:15]:
```suggestion
2. **[GoldPolish](https://github.com/bcgsc/goldpolish)** (aka GoldRush-Edit): polishing the genome
```
I think we should keep this to be consistent with the other steps (ex 1)

**Review by lcoombe — COMMENTED** [review]
Thanks Johnathan! Just a few small things.
* I think we should also update the `goldrush_edit_opts` argument in the makefile - to keep things fully consistent
* In this PR, can we update the btllib version dependencies? With the newest GoldPolish, I believe we need btllib v1.6.2+ (requirements.txt and README)

File: README.md (hunk 1 above)
Comment (jwcodee) [reply README.md:15]: sure

**Comment by jwcodee** [comment]
> Thanks Johnathan! Just a few small things.
> 
> * I think we should also update the `goldrush_edit_opts` argument in the makefile - to keep things fully consistent
> * In this PR, can we update the btllib version dependencies? With the newest GoldPolish, I believe we need btllib v1.6.2+ (requirements.txt and README)

done

**Review by lcoombe — APPROVED** [review]
Looks good - thanks Johnathan!

**Pull request merged**